examples/java/.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
qq: I haven't done Java in a while, are we supposed to check-in the .mvn folder or should we add it to `.gitignore`?

Reply:
Maven wrapper(.mvn, mvnw, mvnw.bat) is supposed to be checked in, so we don't have to add a requirement to have Maven distribution installed.

This is the best practice to stick to the specific maven version and have no requirement for Maven, only JDK is required then.

Reply:
That doesn't include the `.mvn` folder though, only `mvnw` and `mvnw.cmd`? Like, checking in a jar feels weird.

Reply:
`.mvn` actually contains wrapper/maven-wrapper.properties with maven version in there.
`maven-wrapper.jar` is optional to be checked in, so MavenWrapperDownloader.java is able to download this jar each time.

Nevertheless, I see you removed maven wrapper which I am completely fine with.
Thank you for review!